functional_test/scripts/run_rot13_L0toL1.py

```diff
@@ -19,6 +19,6 @@ def doProcess(infiles, outfile):
     infiles = sorted(args[:-1])
     outfile = args[-1]
 
-    print "infiles", infiles
-    print "outfile", outfile
+    print("infiles + ", infiles)
+    print("outfile + ", outfile)
```

Comment:
The plus should be outside the quote here, since it's concatanation...this will put in an extra comma and parens.